Remove uniqueItems from statusFilter

anusha94 opened the pull request:
Fixes #128

JimBugwadia reviewed (changes requested):
Looks like an older version (v0.9.0) of controller-gen was used, can you please try with the latest or v0.12.1 (the one previously used)?

anusha94 commented:
> Looks like an older version (v0.9.0) of controller-gen was used, can you please try with the latest or v0.12.1 (the one previously used)?

Yea I changed back to what was being used, and removed other `go.mod` related changes as well.

Tested by installing this CRD on a kind cluster, and it works fine.

JimBugwadia commented:
/lgtm
/approve
